jenkins/README.adoc

```diff
@@ -841,6 +841,8 @@ The env vars that are used in all of the jobs are as follows:
 |KUBERNETES_MINIKUBE | Will you connect to Minikube? | true
 |REPO_WITH_BINARIES | URL to repo with the deployed jars | http://artifactory:8081/artifactory/libs-release-local
 |REPO_WITH_BINARIES_CREDENTIAL_ID    | Credential ID used for the repo with jars | repo-with-binaries
+|DEPLOYMENT_PROJECT_NAME | Name of the Maven module or Gradle subproject that you want to deploy |
```

Comment:
I think that this should come out of the project. We shouldn't declare it. Maybe it would be enough to provide a property in maven. I'll think about it